Add/università vs self made (#187)
Il-Libro-Open-Source/book: docs/it/università-vs-self-made.md

# Università VS Self-Made

## La terza via: l’ITS

> Nel momento in cui si vuole intraprendere la via della “specializzazione” da developer, una volta concluse le scuole superiori, spesso ci si trova davanti a quello che è un bivio: _Intraprendere la via dell’Università o iniziare un percorso di autoformazione?_

Reviewer: ```suggestion
Nel momento in cui si vuole intraprendere la via della “specializzazione” da developer, una volta concluse le scuole superiori, spesso ci si trova davanti a quello che è un bivio: _intraprendere la via dell’Università o iniziare un percorso di autoformazione?_
```

Reviewer: Avrei comunque approfondito il tema "università" e "auto-formazione", per dare modo di misurarsi con tutte le realtà!

Reviewer 2: Sono d'accordo, ma pensavo di lasciare spazio a qualcun'altro su questi due argomenti, siccome non saprei come approfondirli onestamente

> La risposta non è sempre così scontata, vista anche la soggettività del tema. Nel primo caso stiamo parlando di un percorso che dura minimo tre anni (o cinque, a seconda dell’obiettivo), strutturato e tenuto da docenti, mentre nel secondo di uno la cui durata è ignota e che si fonda esclusivamente sulla nostra organizzazione, disciplina e motivazione.

Reviewer: ```suggestion
La risposta non è sempre così scontata, vista anche la soggettività del tema. Nel primo caso stiamo parlando di un percorso che dura minimo tre anni (o cinque, a seconda dell’obiettivo), strutturato e tenuto da docenti formati e che seguono un piano di studi ben preciso, mentre nel secondo abbiamo una durata potenzialmente ignota e che si basa esclusivamente sulla nostra organizzazione, disciplina e motivazione.
```

> In molti però non sono a conoscenza di una terza via, ovvero degli [ITS (_Istituto Tecnico Superiore_)](https://www.miur.gov.it/web/guest/tematica-its) di programmazione, degli enti solitamente finanziati dalla regione e/o da alcune aziende del settore, il cui scopo è quello di formare nel giro di un paio d’anni delle figure professionali da inserire nel mondo dell’IT.

Reviewer: ```suggestion
Molte persone però non sono a conoscenza di una terza via, ovvero degli [ITS (_Istituto Tecnico Superiore_)](https://www.miur.gov.it/web/guest/tematica-its) di programmazione, degli enti solitamente finanziati dalla regione e/o da alcune aziende del settore, il cui scopo è quello di formare nel giro di un paio d’anni delle figure professionali da inserire nel mondo dell’IT.
```

### Un ibrido tra Teoria e Pratica

> **_“Come è strutturato il percorso formativo di un ITS?”_**

Reviewer: ```suggestion
**_“Com'è strutturato il percorso formativo di un ITS?”_**
```

Per integrare in modo corretto e graduale la “Teoria” e la “Pratica”, il piano formativo standard di un ITS prevede:

> - **Primo Anno**: Prevalentemente dedicato alle **lezioni in aula**, dove avrai l'opportunità di apprendere dalle competenze di **docenti aziendali**<sup>1</sup> e universitari, i concetti, i linguaggi e gli strumenti fondamentali della figura professionale a cui è dedicato il corso.

Reviewer: ```suggestion
- **Primo Anno**: Prevalentemente dedicato alle **lezioni in aula**, dove si ha l'opportunità di apprendere dalle competenze di **docenti aziendali**<sup>1</sup> e universitari, i concetti, i linguaggi e gli strumenti fondamentali della figura professionale a cui è dedicato il corso.
```

>   Inoltre c’è da menzionare che in alcuni ITS viene inoltre offerta la possibilità di prepararsi e sostenere esami per certificazioni esterne quali: [CISCO IT Essentials](https://www.netacad.com/courses/os-it/it-essentials), [ECDL](https://www.aicanet.it/aica/qualita/ecdl-full-standard) e [PET/FIRST/ecc.](https://www.cambridgeenglish.org/exams-and-tests/) per citarne solo alcune.

Reviewer 3: Doppia ripetizione
```suggestion
  Inoltre c’è da menzionare che in alcuni ITS viene offerta la possibilità di prepararsi e sostenere esami per certificazioni esterne quali: [CISCO IT Essentials](https://www.netacad.com/courses/os-it/it-essentials), [ECDL](https://www.aicanet.it/aica/qualita/ecdl-full-standard) e [PET/FIRST/ecc.](https://www.cambridgeenglish.org/exams-and-tests/) per citarne solo alcune.
```

    <br>

> - **Secondo Anno**: Dopo una breve fase di lezioni dedicate agli ultimi moduli, mirate a esplorare concetti più avanzati, come l'_Architettura del Software_ per esempio, l'attenzione si sposterà principalmente sullo **stage lavorativo**, nonché ultima attività prima dell’esame finale (necessario per conseguire un _Diploma di [V livello EQF](https://europa.eu/europass/it/description-eight-eqf-levels)_).

Reviewer: ```suggestion
- **Secondo Anno**: Dopo una breve fase di lezioni dedicate agli ultimi moduli, mirate a esplorare concetti più avanzati, come l'_Architettura del Software_ per esempio, l'attenzione si sposta principalmente sullo **stage lavorativo**, nonché ultima attività prima dell’esame finale (necessario per conseguire un _Diploma di [V livello EQF](https://europa.eu/europass/it/description-eight-eqf-levels)_).
```

<br>

**_“Come posso trovare gli ITS?”_**

> Prima di darti la risposta a questa domanda, tengo a precisare che gli ITS in Italia offrono una vasta gamma di corsi dedicati a settori diversi, tra cui tecnologia, industria, e servizi. Per ovvi motivi in questo libro, più precisamente in questo capitolo, si parlerà di quelli legati al settore IT.

Reviewer 3: Così sembra meno un parere personale

```suggestion
Prima di rispondere a questa domanda, è bene precisare che gli ITS in Italia offrono una vasta gamma di corsi dedicati a settori diversi, tra cui tecnologia, industria, e servizi. Per ovvi motivi in questo libro, più precisamente in questo capitolo, si parlerà di quelli legati al settore IT.
```

Chiusa questa parentesi, il sito dell’**_[INDIRE](https://www.indire.it/progetto/its-istituti-tecnici-superiori/dove-sono-gli-its/)_** (_Istituto Nazionale Documentazione Innovazione Ricerca Educativa_) è sicuramente la miglior risposta a questa domanda. Ovviamente non l’unica, ma sicuramente tra tutte le pagine risulta essere la più completa, presentando anche a sostegno i dati relativi agli ITS della regione e all’occupazione degli studenti diplomati.

<br>

**“_Perché l’ITS potrebbe risultare un’ottima via?”_**

> Innanzitutto come già citato è un corso molto breve (2 anni), a numero chiuso (quindi con un test di ammissione) e alla portata di tutti dal punto di vista economico, dal momento che l’iscrizione è gratuita o a costo molto ridotto. Solitamente viene fornito un PC da poter utilizzare durante le lezioni e dedicato completamente per la programmazione, ma soprattutto, ciò che contraddistingue l’ITS (e il motivo per cui viene citato in questo capitolo) è la sua **dualità tra lezioni in aula e stage in azienda** per poter testare in modo pratico “sul campo” i concetti appresi.

Reviewer 4: ```suggestion
Innanzitutto come già citato è un corso molto breve (2 anni), a numero chiuso (quindi con un test di ammissione) e alla portata di tutti dal punto di vista economico, dal momento che l’iscrizione è gratuita o a costo molto ridotto. Solitamente viene fornito un PC da poter utilizzare durante le lezioni e dedicato completamente alla programmazione, ma soprattutto, ciò che contraddistingue l’ITS (e il motivo per cui viene citato in questo capitolo) è la sua **dualità tra lezioni in aula e stage in azienda** per poter testare in modo pratico “sul campo” i concetti appresi.
```

> È vero, sicuramente la preparazione concettuale risulterà meno solida di quella universitaria, ma è proprio qua che deve subentrare la nostra proattività e la nostra curiosità!

Reviewer: Non credo sia questione di solidità, ma piuttosto di preparazione verso il mondo del lavoro!

> “_Si conclude il modulo di lezioni su Python e vorrei saperne ancora di più. Ottimo! Partendo da questa infarinatura posso apprendere più facilmente anche in maniera autonoma._”

Reviewer: Non ho compreso il senso della frase nel contesto attuale.. forse era un esempio di stereotipi nel settore?

Reviewer 2: Ho scritto questa frase basandomi su diverse testimonianze di colleghi e non. Alcuni si sono laureati mentre altri hanno svolto un corso più simile all'ITS e parlandoci mi è sembrato emergere questo aspetto, ovvero che il corso post-diploma è molto più mirato al mondo lavorativo tralasciando alcuni concetti teorici che invece si trattano all'università

> “_Mi trovo a fare uno stage in un’azienda che usa VUE.js? Conosco JavaScript ma non avevo mai considerato questo Framework. Ecco un’ottima opportunità per farlo ora_”.

Reviewer: Come sopra. Forse aggiungerei una riga o due per dare un filo logico ai paragrafi prima e dopo

> L'importante è alimentare sempre questo “fuoco”, questa sete di conoscenza, che oltre ad essere fondamentale in questo settore (perché non si smette mai di imparare nel mondo IT 🤓) ci aiuterà a crescere come figura professionale e a farci vedere di buon occhio da aziende in cui si potrebbe fare esperienza oltre ad imparare concetti nuovi.

Reviewer 3: Mi sembrava un po' troppo arzigogolata

```suggestion
L'importante è alimentare sempre questo “fuoco”, questa sete di conoscenza, che oltre ad essere fondamentale in questo settore (perché non si smette mai di imparare nel mondo IT 🤓) ci aiuterà a crescere come figura professionale e a farci notare da aziende dove è più facile fare esperienza e assimilare concetti nuovi.
```

È vero, sicuramente la preparazione concettuale risulterà meno solida di quella universitaria, ma è proprio qua che deve subentrare la nostra proattività e la nostra curiosità!

“_Si conclude il modulo di lezioni su Python e vorrei saperne ancora di più. Ottimo! Partendo da questa infarinatura posso apprendere più facilmente anche in maniera autonoma._”

“_Mi trovo a fare uno stage in un’azienda che usa VUE.js? Conosco JavaScript ma non avevo mai considerato questo Framework. Ecco un’ottima opportunità per farlo ora_”.

L'importante è alimentare sempre questo “fuoco”, questa sete di conoscenza, che oltre ad essere fondamentale in questo settore (perché non si smette mai di imparare nel mondo IT 🤓) ci aiuterà a crescere come figura professionale e a farci vedere di buon occhio da aziende in cui si potrebbe fare esperienza oltre ad imparare concetti nuovi.

> 
> A proposito di prime esperienze a livello aziendale, ti invito a leggere l'apposito capitolo “Carriera - Junior” in cui il tema viene trattato in modo più approfondito e sicuramente più stimolante.

Author: ```suggestion
```
